acquia_lift.admin.wizard.inc

```diff
@@ -1030,6 +1030,7 @@ function acquia_lift_personalize_campaign_wizard_submit(&$form, &$form_state) {
 
   // Run the base submit form handling.
   $agent_data = personalize_campaign_wizard_submit_base($form, $form_state);
+  $agent_data = personalize_agent_load($agent_data->machine_name);
   $agent_instance = personalize_agent_load_agent($agent_data->machine_name);
```

Comment:
So the personalize_campaign_wizard_submit_base is supposed to always return the correct agent.  If it's new then it creates it, otherwise it should read it from the form.  I think the problem here is that it calls _personalize_agent_from_form_values to generate the base agent class... but that's actually resetting the data property because its looking for the options value that will no longer exist due to removing optionsForm.  Clearly one of the things I missed when ripping that out.  That function gets called from a couple of places so it should be fixed or we'll start seeing other weirdness I think.